docs/modules/xenome.md

```diff
@@ -0,0 +1,10 @@
+---
+Name: Xenome
+URL: https://github.com/data61/gossamer/blob/master/docs/xenome.md
+Description: >
```

Comment:
The CI is failing to validate this block.

That's because the keys need to be [lowercased](https://multiqc.info/docs/development/modules/#documentation).
```suggestion
name: Xenome
url: https://github.com/data61/gossamer/blob/master/docs/xenome.md
description: >
```

Reply:
Will correct this.